e2e/portalicious/tests/ChangeRegistrationStatus/MovePasFromIncludedToCompleted.ts

```diff
@@ -0,0 +1,81 @@
+import test from '@playwright/test';
+
+import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
+import { doPayment } from '@121-service/test/helpers/program.helper';
+import {
+  changeBulkRegistrationStatus,
+  importRegistrations,
+} from '@121-service/test/helpers/registration.helper';
+import { getAccessToken } from '@121-service/test/helpers/utility.helper';
+import {
+  programIdPV,
+  registrationPvMaxPayment,
+} from '@121-service/test/registrations/pagination/pagination-data';
+
+import { Components, Pages } from '../../../helpers/interfaces';
+
+export default (pages: Partial<Pages>, components: Partial<Components>) => {
+  test('[31211] Move PA(s) from status "Included" to "Completed"', async () => {
+    const accessToken = await getAccessToken();
+    const paymentReferenceId = [registrationPvMaxPayment.referenceId];
+    const { basePage, registrations } = pages;
+    const { tableComponent } = components;
+
+    if (!basePage || !registrations || !tableComponent) {
+      throw new Error('pages and components not found');
+    }
```

Comment:
```suggestion
    const homePage = new HomePage(page);
    const registrations = new RegistrationsPage(page);
    const tableComponent = new TableComponent(page);
```

Following up on my comment above, I don't understand the need for a different paradigm here.

Reply:
In general the idea was to keep the same browser session for all the tests it would be even more beneficial if those tests shared same db setup and were sequential. Unfortunately the more tests I wrote the less it made sense. Especially for validated status that was just becoming included and other statuses becoming harder and harder to follow. Now the setup is doing a reset each time new test is executed so keeping the same browser context does not make sense anymore. If it was different situation and test were sequential and could benefit from same browser context then Playwright needs to pass the same page object for each tests meaning they need to be shared with a helper.

Saying that the benefits are close to none so I'll stick to the current implementation.